chemreg/compound/serializers.py

```diff
@@ -167,15 +168,19 @@ class Meta(DefinedCompoundSerializer.Meta):
             "calculated_inchikey",
         ]
 
+    @catch_compound_calc_error
```

Comment:
Clever use of decorator, but given simplicity of functions, not sure if
```python
try:
    return obj.indigo_structure.molecularWeight()
except IndigoException:
    return None
```
wouldn't be sufficient.  But good framework for error evaluation in general.

Reply:
The error also occurs in  `.grossFormula()` - I wasn't sure if there was also an instance where `.smiles()` and `get_inchikey(...)` would error out that just wasn't covered in the test data. I don't remember if I'd tried a `None` type return on fail.

CI - "very basic catch for FIBr"
![image](https://user-images.githubusercontent.com/74921039/108216417-72821b00-7100-11eb-9f57-3af4098ba6b1.png)


I ran `chemreg/compound/tests/test_views.py`  `def test_definedcompound_detail_attrs(...)...` like 40k times to try and find some bunk compound factory compounds that threw the "bad valence..." error but I was getting some really strange and conflicting results and in the end and the investment v. reward just didn't seem to be there to hunt down what the heck was going on. So I just wrapped it all and kept going.